dbms/src/Core/Protocol.h

```diff
@@ -69,6 +69,7 @@ namespace Protocol
             Totals = 7,               /// A block with totals (compressed or not).
             Extremes = 8,             /// A block with minimums and maximums (compressed or not).
             TablesStatusResponse = 9, /// A response to TablesStatus request.
+            CapnProto = 10,           /// Cap'n Proto
```

Comment:
CapnProto is metadata format name. It could be used not only for defaults but for any sort of meta information and protocols extensions.
We may call it 'Metadata' but there could be another metadata format in future.